assets/js/filter-adr.js

```diff
@@ -0,0 +1,24 @@
+function filterADR(category) {
+  var adrCards = document.querySelectorAll('.adr-card');
+  var filterButtons = document.querySelectorAll('.filter-button');
+
+  adrCards.forEach(function (card) {
+    var categories = card.getAttribute('data-categories').split(' ');
+    if (category === 'All' || categories.includes(category)) {
+      card.style.display = 'block';
+    } else {
+      card.style.display = 'none';
+    }
```

Comment:
How do you feel about replacing this with the ternary operator?

```suggestion
    card.style.display = (category === 'All' || categories.includes(category)) ? 'block' : 'none';
```

Reply:
@hussainweb Done